requirements-common.txt

```diff
@@ -18,6 +18,6 @@ prometheus_client >= 0.18.0
 prometheus-fastapi-instrumentator >= 7.0.0
 tiktoken >= 0.6.0  # Required for DBRX tokenizer
 lm-format-enforcer == 0.10.1
-outlines >= 0.0.43 # Requires torch >= 2.1.0
+outlines == 0.0.43 # Requires torch >= 2.1.0
```

Comment:
Would it also work to do this? Outlines tends to have restrictive dependencies so we should try to avoid it
```suggestion
outlines >= 0.0.43, < 0.1 # Requires torch >= 2.1.0
```

Reply:
I have modified the requirements into the one you have proposed and verified that it is working as well.
<img width="304" alt="image" src="https://github.com/vllm-project/vllm/assets/18331729/b5696bb4-9d25-4ad5-ac80-588f7625633a">